services/storage/tests/test_python_client_sdk.py

```diff
@@ -0,0 +1,38 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+from typing import Dict
+from pathlib import Path
+import pytest
+
+HERE = Path(__file__).resolve().parent
+CLIENT_SDK_DIR = HERE / ".." / "client-sdk" / "python"
+
+
+@pytest.fixture
+def search_dict() -> Dict[str, int]:
+    return {
+        "ClientTimeout(": 0,
+        '"HTTP_CLIENT_REQUEST_AIOHTTP_CONNECT_TIMEOUT", "5"': 0,
+        '"HTTP_CLIENT_REQUEST_AIOHTTP_SOCK_CONNECT_TIMEOUT", "5"': 0,
+    }
+
+
+def assert_all_hits_found(filled_search_dict: Dict[str, int]) -> None:
+    for search_key, value in filled_search_dict.items():
+        assert (
+            value > 0
+        ), f"Could not find an entry for search_key='{search_key}' in storge's python client"
+
+
+def test_are_settings_present(search_dict):
+    # scanning all the files in the
+    all_python_files = CLIENT_SDK_DIR.rglob("*.py")
```

Comment:
you cannot install simcore_sdk and test it instead of parsing the files?

Reply:
This was the faster alternative and solves the issue I need. Avoids people from regenerating this piece of code and forgetting about it.
I've improved it a bit now.